packages/manager-components/package.json

```diff
@@ -27,6 +27,7 @@
     "@ovhcloud/ods-components": "17.2.1",
     "@ovhcloud/ods-theme-blue-jeans": "17.2.1",
     "@tanstack/react-table": "^8.11.6",
+    "@tanstack/react-query": "5.29.2",
```

Comment:
**question:**It is already included on the `peerDependencies`. Is it still required?
```suggestion
```

Reply:
Sure, I delete it 👍